### PR TITLE
Release 0.2

### PR DESCRIPTION
**Pull request by nsheff**
adds CLI
reduces message severity.

### CODE REVIEW

**Comment by vreuter** [comment]
> Actually fixes #21

Man, without an intended comma that comment has some snark

**Comment by nsheff** [comment]
yeah, good point

File: divvy/compute.py [hunk 1]
```diff
@@ -281,3 +282,67 @@ def _handle_missing_env_attrs(self, config_file, when_missing):
...
+    args, remaining_args = parser.parse_known_args()
+
+    keys = [str.replace(x, "--", "") for x in remaining_args[::2]]
+    custom_vars = dict(zip(keys, remaining_args[1::2]))
+    _LOGGER.info("Custom vars: {}", custom_vars)
+
+
+    dcc = ComputingConfiguration(args.config)
+    dcc.activate_package(args.package)
+    yaml_vars = {}
```
Comment (vreuter) [inline divvy/compute.py]: It looks like this line can be removed?

**Comment by vreuter** [comment]
@michalstolarczyk you should add yourself as an author

**Comment by nsheff** [comment]
> > Actually fixes #21
> 
> Man, without an intended comma that comment has some snark

I've learned to ignore your snark :see_no_evil:

**Pull request merged**